src/api/sso/test/e2e/signup-flow.test.js

```diff
@@ -117,35 +115,28 @@ describe('Signup Flow', () => {
       expect(jwt.family_name).toEqual(galileoGalilei.lastName);
       expect(jwt.name).toEqual(galileoGalilei.displayName);
       expect(jwt.roles).toEqual(['seneca', 'telescope']);
-      expect(jwt.picture).toEqual(galileoGalilei.github.avatarUrl);
+      expect(jwt.picture).toEqual(galileoGalilei.githubAvatarUrl);
 
       return { id: jwt.sub, token };
     }
 
+    // TODO:
```

Comment:
Let's delete all this commented code, and @joelazwar can add it back when he writes these tests with the proper JWT authentication for a browser.